nodeup/pkg/model/ntp.go

```diff
@@ -46,22 +49,102 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
+	var ntp string
+	switch b.Cluster.Spec.CloudProvider {
+	case "aws":
+		ntp = "169.254.169.123"
+	case "gce":
+		ntp = "time.google.com "
```

Comment:
Extra whitespace

Reply:
Great spot!! I will fix that now

Reply:
I removed the white space thanks @joshbranham